package.json

```diff
@@ -18,6 +18,7 @@
     "@types/react": "^19.1.13",
     "@types/react-dom": "^19.1.9",
     "astro": "^5.13.7",
+    "clsx": "^2.1.1",
```

Comment:
just a util to make it easier to add condition classes. It's actually the same util Astro components use to let you do:
```tsx
<MyComponent
  class:list={{'is-open': !someVariable, 'with-highlight': someOtherVar }}
/>
```
see [`class:list`](https://docs.astro.build/en/reference/directives-reference/#classlist)